pkg/ui/workspaces/db-console/src/views/reports/containers/network/index.tsx

```diff
@@ -113,17 +148,16 @@ export class Network extends React.Component<NetworkProps, INetworkState> {
   refresh(props = this.props) {
```

Comment:
It looks like this function always uses `this.props`. Can we remove the param and just use `this.props` in the body?